setup.py

```diff
@@ -74,6 +74,7 @@ def local_file(name):
     zip_safe=False,
     extras_require=extras,
     install_requires=install_requires,
+    python_requires='==2.7,>=3.4',
```

Comment:
I don’t think this works (that’s why I didn’t try to exclude 3.3 in #603). Quoting [PEP 440](https://www.python.org/dev/peps/pep-0440/#version-specifiers):

> The comma (`","`) is equivalent to a logical **and** operator: a candidate version must match all given version clauses in order to match the specifier as a whole.

There’s no way in this syntax to allow Python 2.7 or Python 3.4+.